src/components/response/ResponseBlock.tsx

```diff
@@ -53,7 +53,7 @@ export default function ResponseBlock({
 
   const responses = config.response.filter(
     (r) =>
-      (!r.location && location === 'belowStimulus') || r.location === location
```

Comment:
Since location is optional, the default was 'belowStimulus'. This line change breaks that behavior and if a response is declared without location, it will no longer render